Feature/handle accept language (#1)
StudistCorporation/ring-accept-language: README.md

> Ring-Accept-language

Reviewer: I'd use markdown's H1
```suggestion
# Ring Accept-language
```

Author: Thanks, I've fix it.

====

## Overview

> Ring-Accept-language provides a middleware that sets a hash that convert an accept-language in a request header.

Reviewer: ```suggestion
Ring-Accept-language provides a middleware that parses the Accept-language request header and injects the value into the request hash.
```

Author: Thanks, I've fix it.

## Usage

Add a middleware after adding dependencies.

>     (defn some-middleware

Reviewer: Wrap this in code with ```

Author: Thanks, I've fix it.

      [handler]
      (wrap-accept-language handler))

## License

Copyright © 2019 Studist Corporation

> Distributed under the MIT License, the same as Ring.

Reviewer: Confirm with company if MIT License is ok

Author: Thank you for teaching me to write a license.
I've fixed it.
